### PR TITLE
Arxivce 1857 expensive queries

### PR DESCRIPTION
**Pull request by mnazzaro**

### CODE REVIEW

File: #DECISIONS.md# [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (bdc34) [inline #DECISIONS.md#:1]: Delete this temp file.

File: browse/services/database/listings.py [hunk 2]
```diff
@@ -347,10 +347,11 @@ def get_articles_for_month(
                 | (doc.paper_id.like(f"%/{year % 100:02d}%"))
             )                     
   
+    doc_ids_sub = doc_ids.subquery()
     cat_conditions = [and_(aic.c.archive == arch_part, aic.c.subject_class == subj_part) for arch_part, subj_part in cats]
     #filters to only the ones in the right category and records if any of the requested categories are primary
     cat_query = (session.query(aic.c.document_id, func.max(aic.c.is_primary).label('is_primary'))
-        .where(aic.c.document_id.in_(doc_ids))
+        .join(doc_ids_sub, doc_ids_sub.c.document_id == aic.c.document_id)
```
Comment (bdc34) [inline browse/services/database/listings.py:354]: Do we want to make these even minor changes? Or do we just want to time what we have?

**Review by bdc34 — CHANGES_REQUESTED** [review]
Delete temp file

File: browse/services/database/listings.py (hunk 2 above)
Comment (mnazzaro) [reply browse/services/database/listings.py:354]: It passes tests, it's pretty simple logically, and I've tested it pretty extensively in cloud run. It's also just more correct

**Comment by bdc34** [comment]
Mark, Charles and I looked at this in a meeting and decided to just deploy the logging change. There was a bug with the change to  counts.

**Review by bdc34 — APPROVED** [review]

**Pull request merged**